ide/httpserver/src/org/netbeans/modules/httpserver/HttpServerSettings.java

```diff
@@ -428,54 +382,6 @@ Set<String> getGrantedAddressesSet() {
         return addr;
     }
 
-    Properties getMappedServlets() {
-        return mappedServlets;
-    }
-
-    /** Converts string into string that is usable in URL. 
-     *  This mangling changes some characters
-     */
-    static String mangle (String name) {
-        StringBuffer sb = new StringBuffer ();
-        for (int i = 0; i < name.length (); i++) {
-            if (Character.isLetterOrDigit (name.charAt (i)) ||
-                name.charAt (i) == '.') {
-                sb.append (name.charAt (i));
-            }
-            else {
-                String code = Integer.toHexString ((int)name.charAt (i)).toUpperCase ();
-                if (code.length ()<2)
-                    code = (code.length () == 0)? "00": "0"+code;   // NOI18N
-                sb.append ("%").                                    // NOI18N
-                append ((code.length () == 2)? code: code.substring (code.length ()-2));
-            }
-        }
-        return sb.toString ();
-    }
-    
-    /** Unconverts string from URL into old string. 
-     *  This mangling decodes '%xy'
-     */
-    static String demangle (String name) {
-        StringBuffer sb = new StringBuffer ();
```

Comment:
since this file is already touched why not changing this to StringBuilder?

Reply:
Because I stared at this for multiple hours and will only invest more time, if really necessary.